common/build.sh

```diff
@@ -8,7 +8,9 @@ popd > /dev/null
 
```

Comment:
For reviewers: It looks like this file is actually only used for the dev. VM provisioning. Should it be moved to `contrib/vagrant/scripts/`?